src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs

```diff
@@ -0,0 +1,135 @@
+            public static extern void                  Dali_Toolkit_DevelControl_SetAccessibilityReadingInfoType(global::System.Runtime.InteropServices.HandleRef arg1, Tizen.NUI.BaseComponents.View.ReadingInfoTypes arg2);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_new_ReadingInfoType")]
+            public static extern global::System.IntPtr Dali_Toolkit_DevelControl_new_ReadingInfoType();
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_new_GetAccessibilityReadingInfoType")]
+            public static extern global::System.IntPtr Dali_Toolkit_DevelControl_new_GetAccessibilityReadingInfoType(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_delete_ReadingInfoType")]
+            public static extern void                  Dali_Toolkit_DevelControl_delete_ReadingInfoType(Tizen.NUI.BaseComponents.View.ReadingInfoTypes arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_ReadingInfoTypes_Get")]
+            public static extern bool                  Dali_Toolkit_DevelControl_ReadingInfoTypes_Get(Tizen.NUI.BaseComponents.View.ReadingInfoTypes arg1, int arg2);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_ReadingInfoTypes_Set")]
+            public static extern void                  Dali_Toolkit_DevelControl_ReadingInfoTypes_Set(Tizen.NUI.BaseComponents.View.ReadingInfoTypes arg1, int arg2, int arg3);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_ClearAccessibilityHighlight")]
+            public static extern bool                  Dali_Toolkit_DevelControl_ClearAccessibilityHighlight(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_GrabAccessibilityHighlight")]
+            public static extern bool                  Dali_Toolkit_DevelControl_GrabAccessibilityHighlight(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_new_GetAccessibilityStates")]
+            public static extern global::System.IntPtr Dali_Toolkit_DevelControl_new_GetAccessibilityStates(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_new_States")]
```

Comment:
maybe we could add Accessibility to States in function nameson the csharp-binder level?